aws/table_aws_waf_rate_based_rule.go

```diff
@@ -0,0 +1,224 @@
+			ShouldIgnoreError: isNotFoundError([]string{"WAFNonexistentItemException", "ValidationException"}),
+			Hydrate:           getAwsWafRateBasedRule,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listAwsWafRateBasedRules,
+		},
+		Columns: awsColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name for the rule.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "rule_id",
+				Description: "The ID of the Rule.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "metric_name",
+				Description: "The name or description for the metrics for a RateBasedRule.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getAwsWafRateBasedRule,
+			},
+			{
+				Name:        "rate_key",
+				Description: "The field that AWS WAF uses to determine if requests are likely arriving from single source and thus subject to rate monitoring.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getAwsWafRateBasedRule,
+			},
+			{
+				Name:        "rate_limit",
+				Description: "The maximum number of requests, which have an identical value in the field specified by the RateKey, allowed in a five-minute period.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getAwsWafRateBasedRule,
+			},
+			{
+				Name:        "predicates",
+				Description: "The Predicates object contains one Predicate element for each ByteMatchSet, IPSet or SqlInjectionMatchSet object that you want to include in a RateBasedRule.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsWafRateBasedRule,
+				Transform:   transform.FromField("MatchPredicates"),
+			},
+			{
+				Name:        "tags_src",
+				Description: "A list of tags assigned to the Rule.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     listAwsWafRateBasedRuleTags,
+				Transform:   transform.FromField("TagInfoForResource.TagList"),
+			},
+
+			// Standard columns for all tables
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+			},
+			{
+				Name:        "tags",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     listAwsWafRateBasedRuleTags,
+				Transform:   transform.FromField("TagInfoForResource.TagList").Transform(wafRateBasedRuletagListToTurbotTags),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsWafRateBasedRuleAkas,
+				Transform:   transform.FromValue(),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+func listAwsWafRateBasedRules(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("listAwsWafRateBasedRules")
+	// Create session
+	svc, err := WAFService(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+	// List call
```

Comment:
Add new line before comments
```suggestion

	// List call
```